src/generic/animateg.cpp

```diff
@@ -14,8 +14,7 @@
   #pragma hdrstop
 #endif  //__BORLANDC__
 
-#if wxUSE_ANIMATIONCTRL
-
+#if wxUSE_ANIMATIONCTRL 
```

Comment:
```suggestion
#if wxUSE_ANIMATIONCTRL 

```
just to avoid a spurious chunk in the diff.